### ISSUE TITLE
Remove pageaction

### ISSUE THREAD

**Issue by Mte90** [Mte90/Share-Backported#167]
As it is not supported anymore by firefox.

**Comment by Ryuno-Ki** [comment]
Do you have an announcement or something?

**Comment by Mte90** [comment]
Yes I got an email some weeks ago about it but if you are using firefox nightly you will see that they removed with the new design.

**Comment by dag7dev** [comment]
Are you referring to this: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/pageAction?

Because the page is still there and it is working on the bar...

**Comment by Mte90** [comment]
The pageaction are removed from Firefox as feature few release ago and is not the button.
We need to remove the code in the manifest, from the settings and the stuff for that.
